src/loader/lwlocks.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#ifndef TIMESCALEDB_LOADER_LWLOCKS_H
+#define TIMESCALEDB_LOADER_LWLOCKS_H
+
+#define RENDEZVOUS_CHUNK_APPEND_LWLOCK "ts_chunk_append_lwlock"
+
+void ts_chunk_append_shmem_startup(void);
```

Comment:
shouldn't these function names be updated to be lwlock specific not chunk append specific